pkg/collector/python/loader.go

```diff
@@ -91,12 +91,13 @@ func getRtLoaderError() error {
 
 // Load tries to import a Python module with the same name found in config.Name, searches for
 // subclasses of the AgentCheck class and returns the corresponding Check
-func (cl *PythonCheckLoader) Load(config integration.Config) ([]check.Check, error) {
+func (cl *PythonCheckLoader) Load(config integration.Config, instance integration.Data) (check.Check, error) {
+	var tc check.Check
```

Comment:
I tried that, it didn't work

Reply:
https://app.circleci.com/pipelines/github/DataDog/datadog-agent/jobs/202187

Reply:
oh, right, `c.version` requires the concrete type `PythonCheck` instead of the `check.Check` interface (which isn't great design, but no need to address that now).

Then for readability I wouldn't define `tc` and simply keep the `return nil, err` calls